catalyst/dl/runner/gan.py

```diff
@@ -76,7 +77,11 @@ class GanRunner(MultiPhaseRunner):
     Various conditioning types, penalties and regularization (such as WGAN-GP)
     can be easily derived from this class
     """
-    _default_experiment = GanExperiment
+    experiment: GanExperiment
```

Comment:
why not just `_default_experiment`?  And all of the rest - where are you using them?

Reply:
_default_experiment is used to create "new" Runners for new Experiments, during .run_experiment, or something like that customisation
the idea is quite simple - Experiments have the same API, so you don't need to rewrite whole .train method, only on class parameter

Reply:
ok, seems like I didn't notice refactoring in core